src/components/FilterPanel.tsx

```diff
@@ -211,19 +253,27 @@ export default function FilterPanel({
           variant="contained"
           disabled={!filtersChanged}
           onClick={() => {
-            setQueryParams({ minAmount, grades, majors, states, schools });
+            setQueryParams({
+              minAmount,
+              grades,
+              majors,
+              states,
+              schools,
+              ethnicities,
+            });
             onClose();
           }}>
           {t('common:actions.apply')}
         </Button>
         <Button
           disabled={!filtersChanged}
           onClick={() => {
-            setMinAmount(params.minAmount);
-            setGrades(params.grades);
-            setMajors(params.majors);
-            setStates(params.states);
-            setSchools(params.schools);
+            setMinAmount(params.minAmount || 0);
+            setGrades(params.grades || []);
+            setMajors(params.majors || []);
+            setStates(params.states || []);
+            setSchools(params.schools || []);
+            setEthnicities(params.ethnicities || []);
```

Comment:
Prevents site from breaking when `cancel` button is clicked

Reply:
Can you add a comment?